content/java/concepts/strings/terms/getBytes/getBytes.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: '.getBytes()'
+Description: 'Encodes the string into an array of bytes that represent the characters in the string.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`getBytes()`** method encodes the string into an array of bytes that represent the characters in the string.
+
+## Syntax
+
+```pseudo
+byte[] byteArray = str.getBytes();
+```
+
+- `byteArray` is a byte array that stores the byte representation of the string.
+- `str` is the string being converted to bytes.
+
+## Example
+
+The example below demonstrates the use of the `.getBytes()` method:
+
+```java
+// Example.java
+import java.util.*;
+
+public class Example {
+  public static void main(String[] args) {
+    String str = "codeacademy.com";
+    byte[] byteArray = str.getBytes();
+    System.out.println(Arrays.toString(byteArray));
```

Comment:
Can we use a different name for `str` and `byteArray` here? Perhaps `myStr` and `myByteArray` could work. Feel free to adjust.